### PR TITLE
Axis range

### PR DESCRIPTION
**Pull request by FrederikRothe**
Same as axis_range on main tool. No need for the maintainability check here, since Code_Climate will never have it.

### CODE REVIEW

File: run-byoqm-on-tags.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ do
   rm -f "${new_dir}/data_script.sh"
 done
 
-git checkout master
+git checkout main
```
Comment (toffernator) [inline run-byoqm-on-tags.sh]: These changes don't belong in this PR

File: visualization/main.py [hunk 2]
```diff
@@ -70,6 +71,7 @@ def _get_line(
         color="red",
     )
     p.xaxis.major_label_orientation = "vertical"
+    p.y_range = Range1d(0, (int(max(other_qm_ys)) * 1.1))
```
Comment (toffernator) [inline visualization/main.py:74]: Do you have an example of what the graphs look like with and without multiplying by 10%?

Comment (FrederikRothe) [reply visualization/main.py:74]: image

Example with margins.

without buffer the line would follow the "roof" of the graph

Comment (FrederikRothe) [reply visualization/main.py:74]: I'm open to removing this "magic" number, I will just merge now for the sake of the experiment such that I get graphs in bounds.
